### ISSUE TITLE
求一个50系显卡的依赖文件

### ISSUE THREAD

**Issue by wuwei1017** [FunAudioLLM/CosyVoice#1512]
显卡5070ti，系统ubuntu22.04。按照文档安装依赖后，运行时提示：NVIDIA GeForce RTX 5070 Ti with CUDA capability sm_120 is not compatible with the current PyTorch installation.而且无法生成语音。

**Comment by inavy** [comment]
操作系统：Ubuntu 24.04.2 LTS
CUDA 版本：12.8
显卡：NVIDIA GeForce RTX 5070

pip freeze

absl-py==2.3.1
aiofiles==23.2.1
aiohappyeyeballs==2.6.1
aiohttp==3.12.15
aiosignal==1.4.0
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
anyio==4.10.0
async-timeout==5.0.1
attrs==25.3.0
audioread==3.0.1
beautifulsoup4==4.13.4
cachetools==5.5.2
certifi==2025.8.3
cffi==1.17.1
charset-normalizer==3.4.3
click==8.2.1
coloredlogs==15.0.1
conformer==0.3.2
contourpy==1.3.2
cycler==0.12.1
Cython==3.1.2
decorator==5.2.1
deepspeed==0.15.1
diffusers==0.29.0
einops==0.8.1
exceptiongroup==1.3.0
fastapi==0.115.6
fastapi-cli==0.0.4
ffmpy==0.6.1
filelock==3.18.0
flatbuffers==25.2.10
fonttools==4.59.0
frozenlist==1.7.0
fsspec==2024.12.0
gdown==5.1.0
google-auth==2.40.3
google-auth-oauthlib==1.0.0
gradio==5.4.0
gradio_client==1.4.2
grpcio==1.57.0
grpcio-tools==1.57.0
h11==0.16.0
hf-xet==1.1.7
hjson==3.1.0
httpcore==1.0.9
httpx==0.28.1
huggingface-hub==0.34.4
humanfriendly==10.0
hydra-core==1.3.2
HyperPyYAML==1.2.2
idna==3.10
importlib_metadata==8.7.0
inflect==7.3.1
Jinja2==3.1.6
joblib==1.5.1
kaldifst==1.7.14
kiwisolver==1.4.9
lazy_loader==0.4
librosa==0.10.2
lightning==2.2.4
lightning-utilities==0.15.2
llvmlite==0.44.0
Markdown==3.8.2
markdown-it-py==4.0.0
MarkupSafe==2.1.5
matplotlib==3.7.5
mdurl==0.1.2
modelscope==1.20.0
more-itertools==10.7.0
mpmath==1.3.0
msgpack==1.1.1
multidict==6.6.4
networkx==3.1
ninja==1.13.0
numba==0.61.2
numpy==1.26.4
nvidia-cublas-cu12==12.8.3.14
nvidia-cuda-cupti-cu12==12.8.57
nvidia-cuda-nvrtc-cu12==12.8.61
nvidia-cuda-runtime-cu12==12.8.57
nvidia-cudnn-cu12==9.7.1.26
nvidia-cufft-cu12==11.3.3.41
nvidia-cufile-cu12==1.13.0.11
nvidia-curand-cu12==10.3.9.55
nvidia-cusolver-cu12==11.7.2.55
nvidia-cusparse-cu12==12.5.7.53
nvidia-cusparselt-cu12==0.6.3
nvidia-nccl-cu12==2.26.2
nvidia-nvjitlink-cu12==12.8.61
nvidia-nvtx-cu12==12.8.55
oauthlib==3.3.1
omegaconf==2.3.0
onnx==1.16.0
onnxruntime-gpu==1.22.0
openai-whisper==20250625
orjson==3.11.1
packaging==24.2
pandas==2.3.1
pillow==11.3.0
platformdirs==4.3.8
pooch==1.8.2
propcache==0.3.2
protobuf==4.25.0
psutil==7.0.0
py-cpuinfo==9.0.0
pyarrow==18.1.0
pyasn1==0.6.1
pyasn1_modules==0.4.2
pycparser==2.22
pydantic==2.7.0
pydantic_core==2.18.1
pydub==0.25.1
Pygments==2.19.2
pyparsing==3.2.3
PySocks==1.7.1
python-dateutil==2.9.0.post0
python-multipart==0.0.12
pytorch-lightning==2.5.2
pytorch-triton==3.1.0+cf34004b8a
pytz==2025.2
pyworld==0.3.4
PyYAML==6.0.2
regex==2025.7.33
requests==2.32.4
requests-oauthlib==2.0.0
rich==13.7.1
rsa==4.9.1
ruamel.yaml==0.18.14
ruamel.yaml.clib==0.2.12
ruff==0.12.8
safehttpx==0.1.6
safetensors==0.6.2
scikit-learn==1.7.1
scipy==1.15.3
semantic-version==2.10.0
shellingham==1.5.4
six==1.17.0
sniffio==1.3.1
soundfile==0.12.1
soupsieve==2.7
soxr==0.5.0.post1
starlette==0.41.3
sympy==1.13.3
tensorboard==2.14.0
tensorboard-data-server==0.7.2
tensorrt-cu12==10.0.1
tensorrt-cu12-bindings==10.0.1
tensorrt-cu12-libs==10.0.1
threadpoolctl==3.6.0
tiktoken==0.11.0
tokenizers==0.19.1
tomlkit==0.12.0
torch==2.7.1+cu128
torchaudio==2.7.1+cu128
torchmetrics==1.8.1
torchvision==0.22.1+cu128
tqdm==4.67.1
transformers==4.40.1
triton==3.3.1
typeguard==4.4.4
typer==0.16.0
typing_extensions==4.14.1
tzdata==2025.2
urllib3==2.5.0
uvicorn==0.30.0
websockets==12.0
Werkzeug==3.1.3
wetext==0.0.4
wget==3.2
yarl==1.20.1
zipp==3.23.0

**Comment by inavy** [comment]
conda create -n cosyvoice -y python=3.10
conda activate cosyvoice

pip uninstall torch torchvision torchaudio -y
pip install torch==2.7.1 torchvision==0.22.1 torchaudio==2.7.1 --index-url https://download.pytorch.org/whl/cu128

**Comment by luoyuyimeng** [comment]
> conda create -n cosyvoice -y python=3.10 conda 激活 cosyvoice
> 
> pip 卸载 torch torchvision torchaudio -y pip 安装 torch==2.7.1 torchvision==0.22.1 torchaudio==2.7.1 --index-url https://download.pytorch.org/whl/cu128

tensorrt还用10.0.1是不是低了点

**Comment by inavy** [comment]
> > conda create -n cosyvoice -y python=3.10 conda 激活 cosyvoice
> > pip 卸载 torch torchvision torchaudio -y pip 安装 torch==2.7.1 torchvision==0.22.1 torchaudio==2.7.1 --index-url https://download.pytorch.org/whl/cu128
> 
> tensorrt还用10.0.1是不是低了点

tensorrt  是 requirements.txt 指定的版本，没报错，就没升

**Comment by wuwei1017** [comment]
> conda create -n cosyvoice -y python=3.10 conda activate cosyvoice
> 
> pip uninstall torch torchvision torchaudio -y pip install torch==2.7.1 torchvision==0.22.1 torchaudio==2.7.1 --index-url https://download.pytorch.org/whl/cu128

方法可行，感谢。另外安装完之后提示openai-whisper版本过低，升级到最新版之后就没有提示了。
